src/ml_gym/starter.py

```diff
@@ -0,0 +1,81 @@
+from typing import List, Type, Dict, Any
+from ml_gym.blueprints.blue_prints import BluePrint
+from ml_gym.gym.gym import Gym
+from ml_gym.gym.jobs import AbstractGymJob
+from ml_gym.validation.validator_factory import ValidatorFactory
+from datetime import datetime
+from ml_gym.util.logger import QueuedLogging
+from multiprocessing import Queue
+from shutil import copyfile
+import os
+from ml_gym.io.config_parser import YAMLConfigLoader
+
+
+class MLGymStarter:
+
+    class EvaluationMode:
```

Comment:
I am not aware of the purpose of this, since it doesnt seem to be used at all.

Reply:
I renamed it to ValidationMode now. So far we have two: GridSearch and nested CV